ecs-cli/modules/clients/aws/cloudformation/params_test.go

```diff
@@ -70,11 +64,11 @@ func TestAddAndValidate(t *testing.T) {
 
 	clusterValue, exists := cfnParams.nameToKeys[ParameterKeyCluster]
 	if !exists {
-		t.Errorf("Expcted key %s does not exist", ParameterKeyCluster)
+		t.Errorf("Expected key %s does not exist", ParameterKeyCluster)
```

Comment:
lol its kind of sad how many typos we have sitting around in the code; glad you're fixing them :)